src/njoy21/input/POWR/Lib1/Card6/Abs.hpp

```diff
@@ -0,0 +1,12 @@
+struct Abs{
+  using Value_t = std::vector< double >;
```

Comment:
Because of the way that argument::parser manages vectors, it auto-sizes it based on the second value provided anyway.  The check for size is mostly just to ensure that this is behaving as expected.